### PR TITLE
drivers: sensor: ccs811: Fix value conversion

### PR DESCRIPTION
**Pull request by tkln**
The arithmetic performed on the CO2 and VOC measurements is pointless as
the units of the sensor already match those of the API, furthermore the
multiplication will overflow the u32_t with CO2 or VOC readings greater
than 4294 ppm or ppb. This CO2 concentration is achievable by breathing
at the sensor.

Signed-off-by: Aapo Vienamo <aapo.vienamo@iki.fi>

### CODE REVIEW

**Comment by tkln** [comment]
Could you please review this @bogdan-davidoaia

**Comment by tkln** [comment]
Could you review this @lpereira ?

**Comment by pfalcon** [comment]
@Mani-Sadhasivam : FYI

File: drivers/sensor/ccs811/ccs811.c [hunk 1]
```diff
@@ -66,21 +66,18 @@ static int ccs811_channel_get(struct device *dev,
...
-		val->val2 = uval % 1000000;
+		val->val1 = drv_data->voc;
+		val->val2 = 0;
 
 		break;
 	case SENSOR_CHAN_VOLTAGE:
 		/*
-		 * Voltage readings are contained in least
-		 * significant 10 bits in volts
+		 * Raw ADC readings are contained in least significant 10 bits
```
Comment (pfalcon) [inline drivers/sensor/ccs811/ccs811.c]: Why the change of this comment?

Comment (pfalcon) [reply drivers/sensor/ccs811/ccs811.c]: Actually nevermind, the original comment indeed not right.

**Review by pfalcon — APPROVED** [review]

**Comment by pfalcon** [comment]
@tkln : I wonder if you checked that SENSOR_CHAN_VOLTAGE and SENSOR_CHAN_CURRENT contain values in the right units either (should be volts and amperes, per sensor.h).

**Review by Mani-Sadhasivam — APPROVED** [review]
Looks good. Good catch!

**Comment by tkln** [comment]
@pfalcon Yes, I checked those as well. The CCS811_VOLTAGE_SCALE multiplication makes uval microvolts and the register value for current is microamps to begin with.
